networking/openshift_network_security/ovn-k-network-policy.adoc

```diff
@@ -0,0 +1,19 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="about-ovn-k-network-policy"]
+= About OVN-Kubernetes network policy
+include::_attributes/common-attributes.adoc[]
+:context: ovn-k-network-policy
+
+toc::[]
+
+Kubernetes offers two features that users can use to enforce network security. One feature that allows users to enforce network policy is the `NetworkPolicy` API that is designed mainly for application developers and namespace tenants to protect their namespaces by creating namespace-scoped policies. For more information, see xref:../../networking/openshift_network_security/network_policy/about-network-policy.adoc#nw-networkpolicy-about_about-network-policy[About network policy].
+
+The second feature is `AdminNetworkPolicy` which consists of two API: the `AdminNetworkPolicy` (ANP) API and the `BaselineAdminNetworkPolicy` (BANP) API. ANP and BANP are designed for cluster and network administrators to protect their entire cluster by creating cluster-scoped policies. Cluster administrators can use ANPs to enforce non-overridable policies that take precedence over `NetworkPolicy` objects. Administrators can use BANP to setup and enforce optional cluster-scoped network policy rules that are overridable by users using `NetworkPolicy` objects if need be. When used together ANP and BANP can create multi-tenancy policy that administrators can use to secure their cluster.
+
+OVN-Kubernetes CNI in {product-title} implements these network policies using Access Control List (ACLs) Tiers to evaluate and apply them. ACLs are evaluated in descending order from Tier 1 to Tier 3.
+
+Tier 1 evaluates `AdminNetworkPolicy` (ANP) objects. Tier 2 evaluates `NetworkPolicy` objects. Tier 3 evaluates `BaselineAdminNetworkPolicy` (BANP) objects.
+
+image::615_OpenShift_OVN-K_ACLs_0324.png[OVK-Kubernetes Access Control List (ACL)]
+
+If traffic matches an ANP rule, the rules in that ANP will be evaluated first. If the match is an ANP `allow` or `deny` rule, any existing `NetworkPolicies` and `BaselineAdminNetworkPolicy` (BANP) in the cluster will be intentionally skipped from evaluation. If the match is an ANP `pass` rule, then evaluation moves from tier 1 of the ACLs to tier 2 where the `NetworkPolicy` policy is evaluated.
```

Comment:
The `NetworkPolicies` point is one I stumbled on when writing this. I like your clarification, but `NetworkPolicies` is used throughout the repo. I _think_ it might be a combination of the console navigation `NetworkPolicy` > `NetworkPolicies` and some mixups in the API docs. Either way, I like your suggestion and have updated the doc including the tense issue as well.